app/views/expenses/_form.html.haml

```diff
@@ -18,6 +18,8 @@
     = form.label t('category.title')
     .form.form-group
       = form.select :category_id, options_for_select(prepare_data_for_selector(@categories), selected: @expense.category&.id), {}, { class: "form-select" }
+  .field.form-group.has-validation
+    = form.number_field :user_id, id: :expense_user_id, class: "form-control", value: current_user.id, type: :hidden
```

Comment:
Hmm... that's good but I'd rather use the `current_user` directly in the controller, instead of passing that within the form. Even for security reasons it looks more convenient.

Reply:
I think i don;t understand how to user `current_user` directly in controller. I don't have any idea of using it :(